src/health/index.js

```diff
@@ -0,0 +1,224 @@
+import { formatPaginatedDataFromServer } from '../utils/pagination';
+
+/**
+ * @typedef {Object} HealthStatus
+ * @property {string} status A health status of value 'healthy' or 'unhealthy'
+ * @property {string} timestamp  ISO 8601 Extended Format date/time string
+ */
+
+/**
+ * @typedef {Object} HealthAsset
+ * @augments Asset
+ * @property {HealthStatus} health
+ */
+
+/**
+ * @interface PaginatedResponse
+ * @template RecordType
+ * @typedef {Object} PaginatedResponse
+ * @property {Object} _metadata Metadata about the pagination settings
+ * @property {number} _metadata.offset Offset of records in subsequent queries
+ * @property {number} _metadata.totalRecords Total number of asset types found
+ * @property {Array.<RecordType>} records
+ */
+
+/**
+ * @typedef {PaginatedResponse.<HealthAsset>} HealthAssetPaginatedResponse
+ */
+
+/**
+ * @typedef {PaginatedResponse.<HealthStatus>} HealthStatusPaginatedResponse
+ */
+
+/**
+ * Module that provides access to the Contxt Health Service
+ *
+ * @typicalname contxtSdk.health
+ */
+class Health {
+  /**
+   * @param {Object} sdk An instance of the SDK so the module can communicate with other modules
+   * @param {Object} request An instance of the request module tied to this module's audience.
+   * @param {string} [organizationId] The organization ID to be used in tenant url requests
+   */
+  constructor(sdk, request, organizationId = null) {
+    const baseUrl = `${sdk.config.audiences.health.host}/v1`;
+
+    this._baseUrl = baseUrl;
+    this._request = request;
+    this._sdk = sdk;
+
+    this._organizationId = organizationId;
+  }
+
+  /**
+   * The health status option constants
+   * @enum {string}
+   * @static
+   * @example
+   * console.log(Health.GOOD) //healthy
+   * console.log(Health.BAD) //unhealthy
+   */
+  static status = {
+    GOOD: 'healthy',
+    BAD: 'unhealthy'
+  };
+
+  /**
+   * Gets all of an organization's assets and their most recent health status
+   *
+   * API Endpoint: '/:organizationId/assets'
+   * Method: GET
+   *
+   * @param {Object} options
+   * @param {string} [options.organizationId] The organization id that owns the assets. Required if an organization id isn't set on the module instance.
+   * @param {number} [options.limit] The maximum number of records to return
+   * @param {number} [options.offset] The number of records you wish to skip before returning records
+   * @param {string} [options.orderBy] The direction to sort by: "asc" or "desc"
+   * @param {string} [options.sortBy] The field to sort by
+   * @returns {Promise}
+   * @fulfill {HealthAssetPaginatedResponse} Information about all contxt applications
+   * @reject {Error}
+   *
+   * @example
+   * contxtSdk.health
+   *   .getAll({
+   *      organizationId: 'bd900b6e-a319-492f-aa95-9715891b9a83',
+   *      limit: 50,
+   *      offset: 100
```

Comment:
In other places in the SDK, we pass the pagination options as an object as a second argument (when there is a first argument) to the function. Could you update this to continue with that pattern?

Reference:
- https://github.com/ndustrialio/contxt-sdk-js/blob/9797e0c0c3fdfffe5ead1a394258303437bb2aad/src/iot/fieldGroupings.js#L216

- https://github.com/ndustrialio/contxt-sdk-js/blob/117f66548364adf562d5002a74d1506fe7816984/src/events/index.js#L211

- https://github.com/ndustrialio/contxt-sdk-js/blob/e6c1ac572826352afb1ec6383b233c648359b00c/src/assets/assetAttributes.js#L480